src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -448,11 +447,9 @@ protected Object invokeDescribable(String symbol, Object _args) {
             // where this UninstantiatedDescribable is ultimately used, the symbol
             // might be resolved with a specific type.
 
-            // we are returning the NamedArgsAndClosure instead of the UninstantiatedDescribable in order to preserve
-            // the discovered interpolated strings that are stored in the NamedArgsAndClosure.
-            args.uninstantiatedDescribable = ud;
-            return args;
+            return new UninstantiatedDescribableWithInterpolation(symbol, null, args.namedArgs, args.interpolatedStrings);
```

Comment:
```suggestion
            // Note: Declarative relies on an instance of UninstantiatedDescribable being returned here in some cases where there is no Pipeline step involved, e.g. for the `triggers` directive.
            return new UninstantiatedDescribableWithInterpolation(symbol, null, args.namedArgs, args.interpolatedStrings);
```

Reply:
added in b828472cff3abc7ccce59127b6a8a1e5845a2eb3